New/src/content/en/events/embeds/event-map.css

```diff
@@ -0,0 +1,44 @@
+html, body {
```

Comment:
yeah we need them. They get applied inside the iframe and tell it to flex properly